cfgov/unprocessed/css/cf-theme-overrides.less

```diff
@@ -144,10 +144,6 @@
    ========================================================================== */
 
 @grid_box-sizing-polyfill-path: '../js';
-@grid_wrapper-width:            1200px;
-@grid_gutter-width:             30px;
-@grid_total-columns:            12;
-@grid_debug:                    false;
```

Comment:
In the spirit of using the file the way it's intended, I think the better course of action here would be to simply update the `@grid_wrapper-width` variable to the correct number.

OR, remove all variables defined in this file that do not deviate from the Capital Framework default.